### ISSUE TITLE
Dual subtitles don't have background and thus sometimes difficult to see

### ISSUE THREAD

**Issue by gaferzli** [ttop32/MouseTooltipTranslator#187]
It would be great if the option to add background to youtube dual subtitles would exist as it is sometimes difficult to read the white text on light backgrounds.

**Comment by hiep98** [comment]
I agree with you. Currently, you can use Windows opacity to do this instead of background opacity in subtitle options.

**Comment by gaferzli** [comment]
Where would I find the window opacity option? Thanks.

**Comment by gaferzli** [comment]
It worked thanks a lot.
